pydoro/pydoro_tui.py

```diff
@@ -52,14 +53,18 @@ def exit_clicked(_=None):
 
 layout = Layout(container=root_container, focused_element=btn_start)
 
-# Key bindings.
+# Key bindings. These values are set in pydoro_core/config.py.
 kb = KeyBindings()
-kb.add("tab")(focus_next)
-kb.add("s-tab")(focus_previous)
-kb.add("right")(focus_next)
-kb.add("left")(focus_previous)
-kb.add("q")(exit_clicked)
-
+for key, bind in bindings.items():
+    if(key == "focus_next"):
```

Comment:
We can get rid of this if statements if we use another dictionary here that maps function names to functions.

```python
ACTIONS = {"focus_next": focus_next, ...}
```

In python functions are objects too, so we can assign them as a value in a dict.